.replit

```diff
@@ -1,2 +1,10 @@
+run = "npm start"
```

Comment:
```suggestion
run = "npm run start"
```

Not sure why Replit has started to do this but when I import the project now I have to update the run command and it suggests adding "run" to it. Even though it shouldn't actually be needed with `npm start`.

Reply:
Thanks for catching this @lasjorg, and for your suggestion. I'm not sure when this started happening either, but I just checked and I do see the same message you posted in [this comment](https://github.com/freeCodeCamp/freeCodeCamp/issues/46910#issuecomment-1200510844).

While the autocorrection does seem to work, I agree that the experience for learners could be improved.

Let's get this change in and I'll work on an issue to track this update in all the boilerplate repos. And this might also give us the opportunity to update the default branch in many of the repos to `main` and get more of them on the same page.